src/qt3utils/applications/controllers/nidaq_rate_counter.yaml

```diff
@@ -0,0 +1,59 @@
+QT3PLE:
+  ApplicationController:
```

Comment:
I think it is probably better to implement YAML files separately for each application since the logic of the generic "`config_from_yaml`" function will be specific to each application. Having the header be the name of the application should help in ensuring that the purpose of the config files are well-defined. 

Alternatively we could make a "super"-config file which has all hardware and applications in one file, but it'll probably be a bit messy.